tests/test_pagination.py

```diff
@@ -195,10 +195,7 @@ def test_study_filter(self, client, participants,
         resp = json.loads(resp.data.decode('utf-8'))
         assert len(resp['results']) == min(expected_total, 10)
         assert resp['limit'] == 10
-        if '/sequencing-centers' in endpoint:
```

Comment:
I'm not really sure why this exception is here? @parimalak

Reply:
Hmm, I don't know why its there, but it looks like it shouldn't be

Reply:
we added because we can create only one sequencing center with that name

Reply:
I think also the check at line 216 should be removed. Only keep the check thats at line 219 `assert len(ids_seen) == resp['total']`